### PR TITLE
ACMS-1572: Test Acquia CMS with SS 7.0 (with and without) DAM and CKEditor 5

### PR DESCRIPTION
**Pull request by chandan-singh7929**
**Motivation**
Fixes ACMS-1572

**Proposed changes**

**Alternatives considered**

**Testing steps**

**Merge requirements**
- [ ] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
- [ ] Manual testing by a reviewer

### CODE REVIEW

**Comment by chandan-singh7929** [comment]
Note: I have committed below file so that it can be deployed and tested on any of the ode environment, please ignore these files during review. Also the one build is failing due to the same reason.
`composer.json` and `composer.lock`

File: composer.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
     "require": {
         "acquia/acquia-cms-starterkit": "^1.0",
         "drupal/acquia_claro": "^1.3.2",
+        "drupal/core": "^9.5",
```
Comment (apathak18) [inline composer.json:22]: @chandan-singh7929 Isn't this should ^9.5 || ^10?
cc: @vishalkhode1

Comment (chandan-singh7929) [reply composer.json:22]: @apathak18 please refer my note above

Comment (vishalkhode1) [reply composer.json]: @chandan-singh7929 / @apathak18 I've asked Deepak to remove this as this is not required.

**Pull request merged**